src/formats/prometheus_text_format.erl

```diff
@@ -76,6 +77,15 @@ format(Registry) ->
   ok = file:close(Fd),
   Str.
 
+-spec escape_label_value(binary() | iolist() | undefined) -> binary().
```

Comment:
Existing type-spec is a bit strange, this function does not return `binary()` for input `undefined`

Reply:
is it possible to fix in this PR?

Reply:
Looks like my local workspace didn't have GPG signatures setup correctly, so I'll create a new PR anyways with the signed commits. I'll update the spec to the following to more accurately represent the function.
```
-spec escape_label_value(binary() | iolist()) -> binary()
                      ; (undefined) -> no_return().
```